guides/source/testing.md

```diff
@@ -984,8 +1044,8 @@ Sign in helper can be a good example:
 #test/test_helper.rb
 
 module SignInHelper
-  def sign_in(user)
-    session[:user_id] = user.id
+  def sign_in_as(user)
+    post sign_in_url(email: user.email, password: user.password)
```

Comment:
The previous example does not work. We can no longer set session like previously in integration style controller tests. I added it from here https://github.com/rails/rails/issues/23386#issuecomment-192954569 as an alternative to the previous example for new integration style tests.